21034 ep monitor classreset should not create a new instance if current is already empty

VincentBlondeau opened the pull request:
https://pharo.fogbugz.com/f/cases/21034/EPMonitor-class-reset-should-not-create-a-new-instance-if-current-is-already-empty

VincentBlondeau committed:
https://pharo.fogbugz.com/f/cases/21034/EPMonitor-class-reset-should-not-create-a-new-instance-if-current-is-already-empty

VincentBlondeau committed:
https://pharo.fogbugz.com/f/cases/21034/EPMonitor-class-reset-should-not-create-a-new-instance-if-current-is-already-empty

VincentBlondeau committed:
Merge remote-tracking branch 'origin/21034-EPMonitor-classreset-should-not-create-a-new-instance-if-current-is-already-empty' into 21034-EPMonitor-classreset-should-not-create-a-new-instance-if-current-is-already-empty

VincentBlondeau committed:
revert not wanted changes

VincentBlondeau committed:
clean class comment

VincentBlondeau commented on src/Epicea/EpRenameClassRefactoring.class.st:
```diff
@@ -21,10 +21,9 @@ EpRenameClassRefactoring class >> rename: oldName to: newName [
 { #category : #converting }
 EpRenameClassRefactoring >> asRBRefactoring [
-
-	^ RBRenameClassRefactoring new 
```
No it was not. I thought it was an automatic refactoring that was let uncommitted

tinchodias replied:
So what do we proceed? could you remove this change and commit to this branch?

VincentBlondeau committed:
Remove unwanted changes

VincentBlondeau committed:
Update EpRenameClassRefactoring.class.st

VincentBlondeau committed:
Update EpRenameClassRefactoring.class.st

voila

VincentBlondeau committed:
Update EpRenameClassRefactoring.class.st

VincentBlondeau committed:
Update EpMonitor.class.st

Actually the call to self curent initialize current... so I access directly the variable

VincentBlondeau commented:
@tinchodias: Ok for me

tinchodias reviewed (approved):
ok for me

estebanlm commented:
but I hate your formatting ;)
